docs/changelogs/v0.14.md

```diff
@@ -57,3 +61,28 @@ the same binaries are still published under the old name at:
 
 - https://dist.ipfs.io/go-ipfs/
 - https://hub.docker.com/r/ipfs/go-ipfs/
+
+The libp2p [`identify`](https://docs.libp2p.io/concepts/protocols/#identify)  [`agentVersion`](https://github.com/libp2p/specs/blob/master/identify/README.md#agentversion) of Kubo has also been changed from `go-ipfs` to `kubo`.
+
+#### 🎒 `ipfs repo migrate`
+
+This new command allows the you to run the repo migration without starting the daemon.
+
+See `ipfs repo migrate --help` for more info.
+
+#### 🚀 Emoji support in Multibase
+
+Kubo now supports [`base256emoji`](https://github.com/multiformats/multibase/blob/master/rfcs/Base256Emoji.md) encoding in all [Multibase](https://docs.ipfs.io/concepts/glossary/#multibase) contexts. Use it for testing Unicode support, as visual aid while explaining Multiformats, or just for fun:
+
+```console
+$ echo -n "test" | ipfs multibase encode -b base256emoji  -
+🚀😈✋🌈😈
+
+$ echo -n "🚀😈✋🌈😈" | ipfs multibase decode -
+test
+
+$ ipfs cid format -v 1 -b base256emoji bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi
+🚀🪐⭐💻😅❓💎🌈🌸🌚💰💍🌒😵🐶💁🤐🌎👼🙃🙅☺🌚😞🤤⭐🚀😃✈🌕😚🍻💜🐷⚽✌😊
+```
+
+[`/ipfs/🚀🪐⭐💻😅❓💎🌈🌸🌚💰💍🌒😵🐶💁🤐🌎👼🙃🙅☺🌚😞🤤⭐🚀😃✈🌕😚🍻💜🐷⚽✌😊`](https://ipfs.io/ipfs/🚀🪐⭐💻😅❓💎🌈🌸🌚💰💍🌒😵🐶💁🤐🌎👼🙃🙅☺🌚😞🤤⭐🚀😃✈🌕😚🍻💜🐷⚽✌😊)
```

Comment:
I didn't thought it would be worth an entry, so I made it a small surprise, anyway LGTM.

I'll rebase on `release-0.14`